api/services/GithubBuildHelper.js

```diff
@@ -131,4 +131,19 @@ const reportBuildStatus = (build) => {
     });
 };
 
-module.exports = { reportBuildStatus };
+const fetchContent = async (build, path = null) => {
+  if (!build.clonedCommitSha) {
+    throw new Error(`Build or commit sha undefined. Unable to fetch ${path} for build@id=${build.id}`);
+  }
+
+  const site = await Site.findByPk(build.site);
+  if (!site) {
+    throw new Error(`Unable to find a site for build@id=${build.id}`);
+  }
+
+  const accessToken = await loadBuildUserAccessToken(build);
```

Comment:
Let's refactor `loadBuildUserAccessToken` so it doesn't have to query anything. We should have (or are able to get beforehand) the Site, Build and User...